tensorflow/contrib/slim/python/slim/evaluation_test.py

```diff
@@ -235,7 +235,7 @@ def testErrorRaisedIfCheckpointDoesntExist(self):
   def _prepareCheckpoint(self, checkpoint_path):
     init_op = control_flow_ops.group(variables.global_variables_initializer(),
                                      variables.local_variables_initializer())
-    saver = saver_lib.Saver()
+    saver = saver_lib.Saver(write_version=saver_pb2.SaverDef.V1)
```

Comment:
This might be the reason slim evaluation test is failing.

Reply:
@drpngx looks like I am not able to cleanly rollback saver changes on master.
Has there been more changes I should be rolling back, too?

Reply:
I thought it was just https://github.com/tensorflow/tensorflow/pull/14341 and you have these changes.